zerver/management/commands/convert_rocketchat_data.py

```diff
@@ -0,0 +1,45 @@
+import argparse
+import os
+from typing import Any
+
+from django.core.management.base import BaseCommand, CommandError, CommandParser
+
+from zerver.data_import.rocketchat import do_convert_data
+
+
+class Command(BaseCommand):
+    help = """Convert the Rocketchat data into Zulip data format."""
+
+    def add_arguments(self, parser: CommandParser) -> None:
+        dir_help = "Directory containing all the `bson` files from mongodb dump of rocketchat."
+        parser.add_argument(
+            "rocketchat_data_dir", metavar="<rocketchat data directory>", help=dir_help
+        )
+
+        parser.add_argument(
+            "--output", dest="output_dir", help="Directory to write converted data to."
+        )
+
+        parser.formatter_class = argparse.RawTextHelpFormatter
+
+    def handle(self, *args: Any, **options: Any) -> None:
+        output_dir = options["output_dir"]
+        if output_dir is None:
+            raise CommandError("You need to specify --output <output directory>")
+
+        if os.path.exists(output_dir) and not os.path.isdir(output_dir):
+            raise CommandError(output_dir + " is not a directory")
+
+        os.makedirs(output_dir, exist_ok=True)
+
+        if os.listdir(output_dir):
+            raise CommandError("Output directory should be empty!")
+        output_dir = os.path.realpath(output_dir)
```

Comment:
A lot of this code is duplicated between our conversion tools; what do you think about doing a small prep PR to migrate the existing import tools to a cleaner interface?  We can start by using `ZulipBaseCommand` from `zerver/lib/management.py`, and then adding a `self.add_data_export_output_args()` similar to `add_realm_args`, and then call a `selfoutput_dir = self.get_output_dir()` that together replace most of the code from here on up.

(The other bits seem harder to avoid duplicating, and are relatively short in any case).